packages/gem/sentry-sidekiq/latest.json

```diff
@@ -1 +1 @@
-4.2.0.json
\ No newline at end of file
+4.1.2.json
```

Comment:
@HazAT something odd here with Ruby when I ran `make sync-all-links`

Reply:
![image](https://user-images.githubusercontent.com/1633368/107865013-9e677d00-6e30-11eb-8286-ce99a6649688.png)

Reply:
Fixing it on another PR: https://github.com/getsentry/sentry-release-registry/pull/43

Otherwise CI doesn't pass